src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -783,6 +783,204 @@ TIP: If you plan to use another DB version, it is important to get the correspon
 
 Moreover, it is also important to notice that the Scheduler and 3 microservices (*scheduling-api*, *job-planner*, and *proactive-cloud-watch*) have to point to the same scheduler database since they use that one.
 
+==== Example configuration with PostgreSQL
+It is recommanded to use a PostgreSQL database in a ProActive prod environment.
```

Comment:
Configuring ProActive with an external database such as PostgreSQL is recommended in a production environment.